src/lib/scalars/index.ts

```diff
@@ -0,0 +1,25 @@
+import * as NexusSchema from '@nexus/schema'
+import { DateTimeResolver, JSONObjectResolver } from 'graphql-scalars'
+import { GraphQLScalarType } from 'graphql'
+
+export type Scalars = Record<string, GraphQLScalarType>
+
+export const builtinScalars = {
+  Date: new GraphQLScalarType({
+    ...DateTimeResolver,
+    name: 'Date',
```

Comment:
`description`?

Reply:
There's already a `description`. I just needed to override the other one because it was mentioning `JsonObject`